Gems/ScriptCanvas/Code/Include/ScriptCanvas/Grammar/AbstractCodeModel.cpp

```diff
@@ -4501,6 +4503,24 @@ namespace ScriptCanvas
             return true;
         }
 
+        void AbstractCodeModel::ParseLocallyDefinedFunctionCalls()
```

Comment:
do you want to rename this function? as it doesn't do any parse, but just check whether function call node is out of date.